packages/ra-ui-materialui/src/auth/Logout.tsx

```diff
@@ -18,8 +18,10 @@ import { useTranslate, useLogout, useAuthState } from 'ra-core';
  *
  * Used for the Logout Menu item in the sidebar
  */
-export const Logout: FunctionComponent<LogoutProps & MenuItemProps<'li'>> =
-    React.forwardRef(function Logout(props, ref) {
+export const Logout: React.ForwardRefExoticComponent<
+    Omit<MenuItemProps<'li'>, 'ref'> & React.RefAttributes<HTMLLIElement>
```

Comment:
Aren't you missing the `LogoutProps` on this line? :thinking:

Reply:
Good catch, fixed